Readme.md

```diff
@@ -249,11 +250,15 @@ import { AdSettings } from 'react-native-fbads';
 
 AdSettings contains global settings for all ad controls.
 
+#### deviceHashedId
```

Comment:
Not sure. I also was surprised by this name but in audience sdk they really called this [`hashed id`](https://developers.facebook.com/docs/audience-network/testing)